python/lsst/ts/cbp/component.py

```diff
@@ -351,8 +351,10 @@ async def set_park(self, park: int = 0):
         None
 
         """
-        park = bool(park)
-        await self.send_command(f"park={int(park)}")
+        await self.send_command(f"park=1")
+
+    async def set_unpark(self):
+        await self.send_command("park=0")
 
     async def check_cbp_status(self):
         """Checks the status of the encoders.
```

Comment:
What does it mean to "check the status"? Please consider "Read and record the status"

("read", not "reads", for active voice).